protos/follow_me/follow_me.proto

```diff
@@ -11,63 +13,63 @@ option java_outer_classname = "FollowMeProto";
     // Set location of the moving target.
-    rpc SetTargetLocation(SetTargetLocationRequest) returns(SetTargetLocationResponse) {}
+    rpc SetTargetLocation(SetTargetLocationRequest) returns(SetTargetLocationResponse) { option (mavsdk.options.async_type) = SYNC; }
     // Get the last location of the target.
-    rpc GetLastLocation(GetLastLocationRequest) returns(GetLastLocationResponse) {}
+    rpc GetLastLocation(GetLastLocationRequest) returns(GetLastLocationResponse) { option (mavsdk.options.async_type) = SYNC; }
     // Start FollowMe mode.
-    rpc Start(StartRequest) returns(StartResponse) {}
+    rpc Start(StartRequest) returns(StartResponse) { option (mavsdk.options.async_type) = SYNC; }
     // Stop FollowMe mode.
-    rpc Stop(StopRequest) returns(StopResponse) {}
+    rpc Stop(StopRequest) returns(StopResponse) { option (mavsdk.options.async_type) = SYNC; }
 }
 
 // Configuration type.
 message Config {
     // Direction relative to the target that the vehicle should follow
     enum FollowDirection {
         FOLLOW_DIRECTION_NONE = 0; // Do not follow
-        FOLLOW_DIRECTION_FRONT_RIGHT = 1; // Follow from front right
-        FOLLOW_DIRECTION_FRONT_LEFT = 2; // Follow from front left
-        FOLLOW_DIRECTION_FRONT = 3; // Follow from front
-        FOLLOW_DIRECTION_BEHIND = 4; // Folow from behind
+        FOLLOW_DIRECTION_BEHIND = 1; // Follow from behind
+        FOLLOW_DIRECTION_FRONT = 2; // Follow from front
+        FOLLOW_DIRECTION_FRONT_RIGHT = 3; // Follow from front right
+        FOLLOW_DIRECTION_FRONT_LEFT = 4; // Follow from front left
     }
 
-    float min_height_m = 1; // Minimum height for the vehicle
-    float follow_distance_m = 2; // Distance from target for vehicle to follow
+    float min_height_m = 1 [(mavsdk.options.default_value)="8.0"]; // Minimum height for the vehicle in meters (recommended minimum 8 meters)
+    float follow_distance_m = 2 [(mavsdk.options.default_value)="8.0"]; // Distance from target for vehicle to follow in meters (recommended minimum 1 meter)
     FollowDirection follow_direction = 3; // Direction to follow in
-    float responsiveness = 4; // How responsive the vehicle is to the motion of the target
+    float responsiveness = 4 [(mavsdk.options.default_value)="0.5"]; // How responsive the vehicle is to the motion of the target (range 0.0 to 1.0)
 }
 
 // Target location for the vehicle to follow
 message TargetLocation {
-    double latitude_deg = 1; // Target latitude in degrees
-    double longitude_deg = 2; // Target longitude in degrees
-    double absolute_altitude_m = 3; // Target altitude in meters above MSL
-    float velocity_x_m_s = 4; // Target velocity in X axis, in meters per second
-    float velocity_y_m_s = 5; // Target velocity in Y axis, in meters per second
-    float velocity_z_m_s = 6; // Target velocity in Z axis, in meters per second
+    double latitude_deg = 1 [(mavsdk.options.default_value)="NAN"]; // Target latitude in degrees
```

Comment:
I think it's `NaN` (which maybe is bad, I could change it) :thinking:

Reply:
Ok fixed.